### ISSUE TITLE
hashcatify.py seems broken on Python 3.10

### ISSUE THREAD

**Issue by pspacek** [anonion0/nsec3map#26]
Versions:
- f146e6f50f3342a31466f2a0ea6efd9ff5d08a75
- Python 3.10.9

Input: [example.zone.txt](https://github.com/anonion0/nsec3map/files/10880437/example.zone.txt)

Reproducer:
```
$ ./hashcatify.py example.zone example.hashcat
 File "/home/pspacek/w/pkg/nsec3map/git/hashcatify.py", line 6, in <module>
    n3map.hashcatify.main()
  File "/home/pspacek/w/pkg/nsec3map/git/n3map/hashcatify.py", line 41, in main
    sys.exit(hashcatify_main(sys.argv))
  File "/home/pspacek/w/pkg/nsec3map/git/n3map/hashcatify.py", line 33, in hashcatify_main
    out.write(":".join((nsec3_hash, "." + zone, salt, iterations)) 
TypeError: a bytes-like object is required, not 'str'
```

**Comment by anonion0** [comment]
This should be fixed in newer versions, no?
It might have been fixed by #12.

**Comment by pspacek** [comment]
Seems fixed on d145b134742e88dc74579e2252db827a321316c7, thanks!
